pkg/updater/resultstore/resultstore_test.go

```diff
@@ -1132,6 +1132,105 @@ func TestProcessGroup(t *testing.T) {
 	}
 }
 
+func TestGetTestResults(t *testing.T) {
+	cases := []struct {
+		name      string
+		testsuite *resultstore.TestSuite
+		want      []*resultstore.TestCase
+	}{
+		{
+			name: "all tests",
```

Comment:
This is a good case, though I think a few more to cover edge and complicated cases would be good.

Reply:
we should also test other cases: e.g. for example nested testSuites or a testSuite without testCases

Reply:
done!